package.json

```diff
@@ -37,7 +36,8 @@
     "eslint-plugin-standard": "^4.0.1",
     "istanbul": "^0.4.5",
     "mocha": "^7.0.1",
-    "mocha-lcov-reporter": "1.3.0"
+    "mocha-lcov-reporter": "1.3.0",
+    "node-serialize": "0.0.4"
```

Comment:
I don’t like npm audit reporting a vulnerability in memoize-fs. So I would rather declare node-serialize as a peerDependency and document its usage in the docs.

Reply:
It's only devDependency and is used in testing only where we know how we are using it and why. npm audit should report `memoize-fs` as vulnerable when end user is using it.